README.md

```diff
@@ -1,6 +1,34 @@
 IDR Download
 ============
 
-Run:
+The https://hub.docker.com/r/imagedata/download image can be used to acquire
+the original files which were imported into IDR:
+
+        docker run -ti --rm -v /tmp/data:/data imagedata/download idr0001 20151116-verified/JL_120731_S6A /data/JL_120731_S6A
+
+Some exemplary source directories are listed below:
+
+| study   | source                                                                                                      | plate                                       | size   |
+|---------|-------------------------------------------------------------------------------------------------------------|---------------------------------------------|--------|
+| idr0001 | 20151116-verified/JL_120731_S6A                                                                             | JL_120731_S6A                               | # 99G  |
```

Comment:
Are the `#`s in front of the sizes meant to indicate something?

Reply:
I assume they were a leftover of sloppy `vim` work. Pushed a removal.